base/test.jl

```diff
@@ -168,4 +168,20 @@ macro inferred(ex)
     end
 end
 
+#Test equivalence of eigenvectors/singular vectors taking into account possible
```

Comment:
"Taking into account" is not very explicit. How about telling what you said in the PR? Anyways, this should go into the docs, not here.